DSA-Campaign-Uplift-Estimation/src/main/java/com/google/sps/servlets/UserApiServlet.java

```diff
@@ -0,0 +1,61 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.gson.JsonObject;
+import com.google.gson.Gson;
+import java.util.ArrayList;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/userapi")
+public class UserApiServlet extends HttpServlet {
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException{
+
+        JsonObject loginInfo = new JsonObject();
+        UserService userService = UserServiceFactory.getUserService();
+
+        // Store whether user is logged in.
+        Boolean isLoggedIn = userService.isUserLoggedIn();
```

Comment:
Can this be a regular boolean?

Reply:
What do you mean by regular Boolean?

Reply:
Just 'boolean'

Reply:
+1 for using boolean instead of Boolean.

Reply:
Fixed.